pybamm/input/parameters/lithium_ion/Ai2020.py

```diff
@@ -0,0 +1,692 @@
+import pybamm
+import os
+
+
+def graphite_diffusivity_Dualfoil1998(sto, T):
+    """
+    Graphite diffusivity as a function of stochiometry [1, 2, 3].
+
+    References
+    ----------
+     .. [1] Ai, W., Kraft, L., Sturm, J., Jossen, A., & Wu, B. (2020).
+     Electrochemical Thermal-Mechanical Modelling of Stress Inhomogeneity in
+     Lithium-Ion Pouch Cells. Journal of The Electrochemical Society, 167(1), 013512
+      DOI: 10.1149/2.0122001JES.
+     .. [2] Rieger, B., Erhard, S. V., Rumpf, K., & Jossen, A. (2016).
+     A new method to model the thickness change of a commercial pouch cell
+     during discharge. Journal of The Electrochemical Society, 163(8), A1566-A1575.
+
+    Parameters
+    ----------
+    sto: :class:`pybamm.Symbol`
+        Electrode stochiometry
+    T: :class:`pybamm.Symbol`
+        Dimensional temperature, [K]
+
+    Returns
+    -------
+    :class:`pybamm.Symbol`
+        Solid diffusivity [m2.s-1]
+    """
+    D_ref = 3.9 * 10 ** (-14)
+    E_D_s = 5000
+    T_ref = pybamm.Parameter("Reference temperature [K]")
+    arrhenius = pybamm.exp(E_D_s / pybamm.constants.R * (1 / T_ref - 1 / T))
+    return D_ref * arrhenius
+
+
+def graphite_electrolyte_exchange_current_density_Dualfoil1998(
+    c_e, c_s_surf, c_s_max, T
+):
+    """
+    Exchange-current density for Butler-Volmer reactions between graphite and LiPF6 in
+    EC:DMC.
+
+    References
+    ----------
+    .. [2] http://www.cchem.berkeley.edu/jsngrp/fortran.html
+
+    Parameters
+    ----------
+    c_e : :class:`pybamm.Symbol`
+        Electrolyte concentration [mol.m-3]
+    c_s_surf : :class:`pybamm.Symbol`
+        Particle concentration [mol.m-3]
+    c_s_max : :class:`pybamm.Symbol`
+        Maximum particle concentration [mol.m-3]
+    T : :class:`pybamm.Symbol`
+        Temperature [K]
+
+    Returns
+    -------
+    :class:`pybamm.Symbol`
+        Exchange-current density [A.m-2]
+    """
+    m_ref = (
+        1 * 10 ** (-11) * pybamm.constants.F
+    )  # (A/m2)(mol/m3)**1.5 - includes ref concentrations
```

Comment:
I think this comment is wrong everywhere and has been forever haha. Shouldn't the units be (A/m2)(m3/mol)**1.5 or equivalent?

Reply:
I agree with Rob. I think it makes sense to leave the units, corrected though, as a quick sanity check.